fixtures/cloudflare-dev-module-resolution/vite.config.ts

```diff
@@ -0,0 +1,42 @@
+import { defineConfig, type ViteDevServer } from 'vite';
+import {
+  cloudflareEnvironment,
+  type DevEnvironment,
+} from '@flarelabs-net/vite-environment-provider-cloudflare';
+import type * as http from 'node:http';
+
+const ssrEnvName = 'ssr-env';
+
+export default defineConfig({
+  appType: 'custom',
+  ssr: {
+    target: 'webworker',
+  },
+  optimizeDeps: {
+    include: [],
+  },
+  plugins: [
+    ...cloudflareEnvironment(ssrEnvName),
+    {
+      name: 'cloudflare-dev-module-resolution-fixture',
+      async configureServer(server: ViteDevServer) {
+        const devEnv = server.environments[ssrEnvName] as DevEnvironment;
+
+        let handler = await devEnv!.api.getHandler({
+          entrypoint: './entry-workerd.ts',
+        });
```

Comment:
Let's throw an error if the devEnv doesn't exist rather than asserting it:

```suggestion
        const devEnv = server.environments[ssrEnvName] as DevEnvironment;
        
        if (!devEnv) {
          throw new Error('Dev environment not found');
        }

        let handler = await devEnv.api.getHandler({
          entrypoint: './entry-workerd.ts',
        });
```